supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -137,19 +137,21 @@ If available, inform users of alternative capabilities and workarounds.
 [discrete]
 ===== Deprecation notice
 ----
-In <product> <release>, <name of capability> is deprecated. Bug fixes and support are provided through the end of the <releasename or, if unknown, “a future”> life cycle. After which, no new feature enhancements are made. (Optional: You can use <alternative capability> instead.)
+In <product> <release>, <name of capability or feature> is deprecated and is planned to be removed in the <deprecation time frame>. Bug fixes and support are provided through the end of the current release life cycle, but be aware that deprecated items do not receive further enhancements and will be removed. [Optional, if an alternative is available] Alternatively, you can use <alternative capability or feature> instead.
 ----
+When citing deprecation time frames, refer to specific releases (such as "the next release") only if that time frame is known to be accurate. Otherwise, refer to "a future release" because it gives flexibility to account for changes to the planned deprecation.
```

Comment:
IBM SG (pg.231) states:

> Future product releases: Avoid claims that are related to the content or timing of a future product or release.

SO should the line with "a future release" be removed?

Reply:
So the release notes are an exception to talking about the future:
https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future

The double quotation thing just came up in a PR I did and for consistency with this guide, we kept them in so I'd suggest the same here. Probably should think about this across the guide at some point though, good point.

Reply:
OK. Thanks, @ndeevy . Must be a product-specific allowance, because I cannot use such language for Runtimes products. I think we got caught out once or twice with such comments that were never delivered.

I'll mark this comment as resolved.